com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/helpers/ApplicationColorDetector.java

```diff
@@ -69,7 +68,7 @@ public ApplicationColor detectLiveApplicationColor(DeployedMta deployedMta, Stri
         if (latestDeployedColor == null) {
             return olderApplicationColor;
         }
-        return phase == Phase.UNDEPLOY ? latestDeployedColor : olderApplicationColor;
+        return phase == Phase.UNDEPLOY ? latestDeployedColor : latestDeployedColor.getAlternativeColor();
```

Comment:
This should be in a separate commit, because it addresses a different issue.

Reply:
done